presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -169,19 +172,57 @@ See also :doc:`/language/timestamp`
 
     Example: ``TIME '01:02:03.456 America/Los_Angeles'``
 
+.. _timestamp-data-type:
+
 ``TIMESTAMP``
 ^^^^^^^^^^^^^
 
     Instant in time that includes the date and time of day without a time zone.
     Values of this type are parsed and rendered in the session time zone.
+    Equivalent to ``TIMESTAMP WITHOUT TIME ZONE``, ``TIMESTAMP(3)``, and
+    ``TIMESTAMP(3) WITHOUT TIME ZONE``.
 
     Example: ``TIMESTAMP '2001-08-22 03:04:05.321'``
 
+``TIMESTAMP(P) WITHOUT TIME ZONE``
```

Comment:
Change this to just ``TIMESTAMP(P)``. We can mention in the description that this type can also be written as ``TIMESTAMP WITHOUT TIME ZONE``. However, the canonical name for the type is just ``TIMESTAMP``, when rendered by the SQL engine (e.g., in ``DESCRIBE``) output, so we should call it that here.

Reply:
Also, from purely human (non-scientific) perspective, `TIMESTAMP WITHOUT TIME ZONE` looks very similar to `TIMESTAMP WITH TIME ZONE` (eg has levenshtein distance of 3 with length of 27, being 88% similar).  `TIMESTAMP` is much easier to read.

Reply:
Yes, that bit me a few times when writing the code to support these types. It's easy to get lost in a sea of names when you have things like`TimestampWithoutTimeZoneToTimestampWithTimeZoneCast`. (I didn't go that route, fwiw)